commit 3c6727ae67ef3c756ab17b500a5bdfe49f5cadc2 (refs/changes/14/3514/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-02-27T19:41:19-08:00 (1 year, 7 months ago)
    
    TRITON-193 TRITON-185 broke sapi startup: "Uncaught Error: ENOENT: no such file or directory, open '/opt/smartdc/sapi/lib/server/../../package.json'"

diff --git a/Makefile b/Makefile
index 48c0103..5a37add 100644
--- a/Makefile
+++ b/Makefile
@@ -95,6 +95,7 @@ service: all $(SMF_MANIFESTS)
 	@touch $(SVC_PKGDIR)/site/.do-not-delete-me
 	cp -r $(TOP)/server.js \
 		$(TOP)/node_modules \
+		$(TOP)/package.json \
 		$(SVC_INSTDIR)/
 	cp -r $(TOP)/lib/common \
 		$(TOP)/lib/server \
